web/packages/teleport/src/services/userPreferences/discoverPreference.ts

```diff
@@ -0,0 +1,90 @@
+/**
+ * Teleport
+ * Copyright (C) 2025  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * Used to uniquely identify resource guides. These UID's will
+ * be stored in the backend as user preference to preserve
+ * which resource guides user wanted to "pin".
+ *
+ * There is no specific format to use, just ensure that enum values
+ * are unique.
+ *
+ * Existing enum values must not be modified.
+ */
+export enum DiscoverGuideId {
```

Comment:
nit: In favor of shorter names (since this will be used to pin the resource internally and not exposed to users right? 

- `Application` can be `App`
- `ApplicationSaml` can be `Saml`
- `Database` can be `Db

Applied to both key and values.

Feel free to ignore if you think we gain by spelling out loud.

Reply:
keeping it as is